src/Plugin/WebformElement/CivicrmSelect.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Drupal\webform_civicrm\Plugin\WebformElement;
+
+use Drupal\Component\Utility\NestedArray;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Render\Markup;
+use Drupal\webform\Plugin\WebformElementBase;
+use Drupal\webform\WebformSubmissionInterface;
+
+/**
+ * Provides a 'civicrm_select' element.
+ *
+ * @WebformElement(
+ *   id = "civicrm_select",
+ *   label = @Translation("CiviCRM Select"),
+ *   description = @Translation("Provides a CiviCRM powered select list."),
+ *   category = @Translation("CiviCRM"),
+ * )
+ *
+ * @see \Drupal\webform_example_element\Element\WebformExampleElement
+ * @see \Drupal\webform\Plugin\WebformElementBase
+ * @see \Drupal\webform\Plugin\WebformElementInterface
+ * @see \Drupal\webform\Annotation\WebformElement
+ */
+class CivicrmSelect extends WebformElementBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getDefaultProperties() {
+    return parent::getDefaultProperties() + [
+        'form_key' => NULL,
+        'pid' => 0,
+        'value' => '',
+        'empty_option' => '',
+        'empty_value' => '',
+        'options' => [],
+        'options_randomize' => FALSE,
+        'expose_list' => TRUE,
+        'exposed_empty_option' => '- ' . t('Automatic') . ' -',
+        'civicrm_live_options' => 1,
+        'extra' => [
+          'civicrm_live_options' => 1,
```

Comment:
Or maybe _this_ is where that extra bit in the Yaml is from?

Reply:
The `extra` can be blown away, now. That was from the D7 work and has now been migrated completely into the element property `civicrm_live_options`